perftests/dynamodb_kv/main.c

```diff
@@ -144,6 +144,207 @@ main(int argc, char * argv[])
 		exit(1);
 	}
 
+	/* Send (faulty) ICAS request. */
```

Comment:
Are you sure that you want to be sending faulty requests in the perftests?  I would have thought that the regular tests would be the best place for that.

Reply:
I don't think we have a regular test for `dynamodb-kv`?  I think this could be turned into one though.

Reply:
I think I might have made this a perftest simply because I didn't want to bother with automatically deciding whether the output was correct and turning it into PASSED or FAILED.

Reply:
`tests/kvlds-ddbkv` uses the binary, at least.  Granted, that doesn't provide a nice place to shove extra C code.  I was also a bit confused between `bench/` and `perftests/`.

If you want to move forward with this now and table the question of potentially moving this code into a regular test, I certainly have no objection.

Reply:
Right, but `kvlds-ddbkv` tests the entire stack, not just `dynamodb-kv`.

"perftests" just means "test code where Colin looks at the output" as opposed to "test code which runs automatically and yields success/failure".  But I don't think there's any reason why we can't automate the testing of `dynamodb-kv`.